spec/models/user_spec.rb

```diff
@@ -11,4 +53,18 @@
     it { expect(subject).to validate_presence_of(:first_name) }
     it { expect(subject).to validate_presence_of(:last_name) }
   end
+
+  describe 'subscribed' do
+    it 'returns true with an active subscription' do
```

Comment:
Instead of having numbered users and subscriptions, you should create different contexts and override `let!(:user)` and `let!(:subscription)` on each context when needed.